### ISSUE TITLE
Taming Asynchronous Programming with RxJS

### ISSUE THREAD

**Issue by shakyShane** [lnug/speakers#60]
Asynchronous programs can be confusing, hard to follow, difficult to debug and, quite frankly, just down right *annoying* to write. 

Mouse & touch events in the browser, XHR requests, node's various event emitters... - what if there was a way to model all of these eventual values in the exact same way? RxJS allows us to do exactly this with the Observable. Boasting a rich set of composable operators, you can begin to unravel your spaghetti code and author declarative transformation pipelines that are simply not possible otherwise.

**Comment by shakyShane** [comment]
An introduction to RxJS, this talk will first cover the high-level concepts of the Observable, followed by some practical examples showing the types of problem that can solved with this approach

**Comment by iancrowther** [comment]
what size t-shirt would you like?

**Comment by shakyShane** [comment]
Small please!
